components/script/dom/htmlscriptelement.rs

```diff
@@ -294,10 +293,11 @@ fn fetch_a_classic_script(
     }));
 
     let (action_sender, action_receiver) = ipc::channel().unwrap();
+    let (task_source, canceller) = doc.window().task_manager().networking_task_source_with_canceller();
```

Comment:
Just found one small thing. I found the latter variable is sometimes named as `canceller` and sometimes named as `task_canceller`.

How about let's keep using a consistent naming? (personally, I think `task_canceller` is much easier to know what it is)